### DISCUSSION TITLE
your_auth_key - What is it?

### DISCUSSION THREAD

**Discussion by liorphp** [norkunas/onesignal-php-api#158] [Q&A] [answered]
`$config = new Config('your_application_id', 'your_application_auth_key', 'your_auth_key');`

What is the third parameter (your_auth_key)? Where should I get it from?

**Comment by norkunas** [comment, marked as answer]
You can find it in `Account & API Keys` section in [app.onesignal.com](https://app.onesignal.com/).

Click on the `account menu` in the right side of the header -> click `Account & API Keys` -> scroll down to `User Auth Key`.

Basically it allows you to manage your apps via API, while with the `app_id` and `app_key` you can only manage only provided application devices and notifications.

![image](https://user-images.githubusercontent.com/2722872/140458299-49d3d4e3-faea-4bd5-a773-05597c100e8d.png)

Also note that it's an optional configuration parameter and you can leave it out as null if you don't need to access to apps api.

**Reply by liorphp** [reply]
Excellent.
Thank you!
